### PR TITLE
YSHOP2-864: Create a reusable breadcrumps partial

### PR DESCRIPTION
**Pull request by adildev101**
## JIRA Ticket

[YSHOP2-864](https://youcanshop.atlassian.net/browse/YSHOP2-864)

## QA Steps

-  [ ] `pnpm i`
-  [ ] `pnpm dev`

## Note

Leave empty when you have nothing to say

[YSHOP2-864]: https://youcanshop.atlassian.net/browse/YSHOP2-864?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: assets/main.css [hunk 1]
```diff
@@ -666,6 +666,10 @@ input:not([type=radio]),
   font-size: 16px;
   text-decoration: none;
 }
+.breadcrumbs-container .breadcrumbs-listings .third-listing {
+  font-weight: 700;
```
Comment (eihabkhan1) [inline assets/main.css:670]: Shouldn't this be 400?

**Review by eihabkhan1 — COMMENTED** [review]
If I use this breadcrumbs component in "cart" page for example I would have to add an `if` statement similar to the one you added in `line 6` in `breadcrumbs.liquid`, right?

File: assets/main.css (hunk 1 above)
Comment (adildev101) [reply assets/main.css:670]: it should be bold than the second listing

**Comment by adildev101** [comment]
> If I use this breadcrumbs component in "cart" page for example I would have to add an `if` statement similar to the one you added in `line 6` in `breadcrumbs.liquid`, right?

Yes right 👍

File: assets/main.css (hunk 1 above)
Comment (eihabkhan1) [reply assets/main.css:670]: In the cod-theme design true, but according to cod-theme-2 it should be 400

File: styles/main.scss [hunk 2]
```diff
@@ -720,7 +720,7 @@ input:not([type='radio']),
     }
 
     .third-listing {
-      font-weight: 700;
+      font-weight: 600;
```
Comment (eihabkhan1) [inline styles/main.scss:723]: What if we only had two listings? Would the second listing be bold as well?

**Review by eihabkhan1 — CHANGES_REQUESTED** [review]

**Review by eihabkhan1 — APPROVED** [review]
Nice 👌

**Pull request merged**